webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
Oh yeah, the legacy junk.  I wouldn't bother fixing that one.  Leave it nullable.

You could "fix" it by adding overloads, but that's just going to cause more work for us to dress up an API point we ultimately want to remove.